pkg/util/trace.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"contrib.go.opencensus.io/exporter/ocagent"
+	log "github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+)
+
+// InitializeTracing initiates trace, exporter and the sampler
+func InitializeTracing(serviceName string, address string, probability float64) {
+	if address != "" {
+		oce, err := ocagent.NewExporter(
+			ocagent.WithInsecure(),
+			ocagent.WithAddress(address),
+			ocagent.WithServiceName(serviceName))
+		if err != nil {
+			log.Errorf("Couldn't create a OC Agent exporter:%s", err)
```

Comment:
If `err != nil`, I don't think we want to call `trace.RegisterExporter(oce)` later, right? (I don't think `log.Errorf()` stops the execution.) [`oce` will be `nil`](https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/blob/a8a6f458bbc1d5042322ad1f9b65eeb0b69be9ea/ocagent.go#L98-L104), which might be [problematic for `trace.RegisterExporter()`](https://github.com/census-instrumentation/opencensus-go/blob/3b5a343282fe4b4fccdb0f24cbd1d7169d20858a/trace/export.go#L53). I think we should just return `err` and let `main` deals with it.